rule_schema_v2.atd

```diff
@@ -352,6 +352,9 @@ type formula = {
 
   (* NEW: since 1.74 *)
   ?fix: string option;
+
+  (* NEW: since 1.79 *)
+  ?as <ocaml name="as_">: string option;
```

Comment:
maybe a comment saying must be a metavariable would be useful. We actually have an mvar type alias we could
use here instead of string.